src/test/java/org/kie/processmigration/persistence/DDLScriptsTest.java

```diff
@@ -98,12 +99,13 @@
     @Inject
     Scheduler scheduler;
 
+    @Inject
+    Flyway flyway;
+
     @AfterEach
-    @Transactional
     void cleanUp() {
-        MigrationReport.deleteAll();
-        Migration.deleteAll();
-        Plan.deleteAll();
+        flyway.clean();
+        flyway.migrate();
```

Comment:
This will execute the SQL scripts, right?

Reply:
correct, based on the prefix, it will pick up and execute one set or another, as configured in the `maven-antrun-plugin` here https://github.com/kiegroup/process-migration-service/blob/e7b9c95f80fe01d6de744953b87b686b14cde59b/pom.xml#L618-L661